src/module.c

```diff
@@ -188,6 +188,11 @@ static int parseRangeArguments(RedisModuleCtx *ctx, Series *series, int start_in
 static int parseCountArgument(RedisModuleCtx *ctx, RedisModuleString **argv, int argc, long long *count) {
     int offset = RMUtil_ArgIndex("COUNT", argv, argc);
     if (offset > 0) {
+        if (strcasecmp(RedisModule_StringPtrLen(argv[offset - 1], NULL), "AGGREGATION") == 0) {
+            int second_offset = offset + 1 + RMUtil_ArgIndex("COUNT", argv + offset + 1, argc - offset - 1);
```

Comment:
@ashtul you can just check if COUNT is before or after AGGREGATION, if its after AGGREGATION it should be more than 2 offsets.

Reply:
@danni-m it can be both before or after or twice after. We use COUNT twice

Reply:
@danni-m It came out nice!
Thanks